core/src/tangram.cpp

```diff
@@ -187,11 +194,65 @@ void render() {
 
             style->onBeginDrawFrame(*m_view, *m_scene);
 
-            // Loop over all tiles in m_tileSet
             for (const auto& tile : m_tileManager->getVisibleTiles()) {
-                tile->draw(*style, *m_view);
+
+                const auto& tileID = tile->getID();
+
+                if (tile->getProxyCounter() == 0) {
+                    RenderState::stencilFunc(GL_ALWAYS, 1, 0xFF);
+                    tile->draw(*style, *m_view);
+                }
+            }
+            style->onEndDrawFrame();
+        }
+
+        // 2. Disable Color buffer writting and invert stencil value wherever depth from proxy wins
+        RenderState::colorWrite(GL_FALSE, GL_FALSE, GL_FALSE, GL_FALSE);
+        RenderState::stencilTest(GL_TRUE);
+        RenderState::stencilWrite(0xFF);
+        RenderState::stencilOp(GL_KEEP, GL_KEEP, GL_INVERT);
+        RenderState::polygonOffsetFill(GL_TRUE);
+        RenderState::polygonOffset(1.0, 1024.0);
```

Comment:
Can you explain the value `1024` for the units param; I'm not sure to understand. It seems that `RenderState::polygonOffset(1.0, 1.0);` should be fine for all depth slope (polygon facing and not facing the camera) refering to http://www.glprogramming.com/red/chapter06.html#name4.